docs/tutorials/comparison.md

```diff
@@ -79,302 +123,753 @@ source1 = "FirstSourceName"
-- Tick the `public` box to make the comparison available on the slow.pics homepage.
-- Name the collection appropriately for Slow.pics, get the TMDB ID from <https://www.themoviedb.org/> and paste it in the TMDB box, with it correctly set to Movie/TV depending on the content.
-- Set the tag to the same name as listed on TMDB.
-- Set "Delete After" to your desired value, by default comps expire after 2 years of not being viewed.
-- Under `random` you may specify an additional frame amount for VS-Preview to pick at random, recommended to help fill the comparison with more frames.
-- Untick `Current frame` as you likely don't want that included.
-- Make sure your PNG compression is set to 0 (Max) under Settings -> Main
+Sets the frame type to capture. *This is a setting used when creating [automatic](#automatic-badge-variant-secondary-text-fastest-option)/[semi-automatic](#semi-automatic-badge-icon-variant-primary-text-recommended) comparisons.*
 
-**If you wish to upload to a slow.pics account for comp history and the ability to edit comps do the following:**
+Only checked frames are captured. For instance, if `I` is unchecked, then VSPreview will only capture `P` or `B` frames.
 
-- Make sure you're logged into slow.pics
-- Open the developer menu, go to Application at the top, then Storage on the left, open Local Storage and select slow.pics. Copy the browserId.
-- Still in the developer menu, go to Network, refresh the page to populate the list, select the slow.pics entry and scroll until you see SLPSESSION and copy the ID.
-- Back in VS-Preview, go to settings, Comp, and paste the two IDs in the relevant boxes.
+==- 4 - TheMovieDB info
 
-### QOL Changes
+The TMDB ID found at [TheMovieDB](https://www.themoviedb.org).
 
-- **Jump through frames quickly**
+- For TV shows, set the box to `TV`
+- For movies, set the box to `Movie`
 
-   In VS-Preview on the bottom bar select Playback, directly right of the playback keys set the `1` value to `120`.
+==- 5 - Collection tags
 
-- **Mark screenshots quicker by setting the mark frame button to enter**
+Tags for Slowpoke Pics collections.
 
-    Open `%localappdata%\Programs\Python\Python311\Lib\site-packages\vspreview\toolbars\comp` and edit `toolbar.py`
+==- 6 - Slowpoke Pics parameters
 
-    - Line 552: Replace `QKeyCombination(Qt.Modifier.CTRL, Qt.Key.Key_Space).toCombined(), self.add_current_frame_to_comp` with `(Qt.Key_Return), self.add_current_frame_to_comp`
+Parameter      | Description
+---------------|--------------------------------------------------------------------------------------------------------------------------
+`Public`       | Sets whether the comparison link is publicly visible on the front page and search on [Slowpoke Pics](https://slow.pics)
+`NSFW`         | Sets whether the comparison displays an NSFW warning before showing images
+`Delete After` | Sets the number of days for when the comparison will expire and be deleted from [Slowpoke Pics](https://slow.pics)
```

Comment:
might be worth mentioning, when unspecified, slow.pic's default is that comps get deleted after 2 years of completely inactivity (inactivity here means that no one has visited the comp in 2 years)